README.md

```diff
@@ -250,6 +250,16 @@ The [quickstart instructions](#quickstart) above include details on how to
 begin exploring your local dev Arcwell instance.
 
 
+## Building for Production
+
+The [quickstart instructions](#quickstart) and guide to this point have been
+focused on building a _development_ environment. Arcwell can be built for
+production-like environments (e.g., your hosted staging, QA/dev, or production
+targets).
+
+Learn more about generated environment-specific, production-like builds within
+the [production build docs](doc/builds.md).
+
```

Comment:
I took a crack at editing this paragraph to reflect the goal of showing we offer production-ready builds and linking out to a special `builds.md` with further instructions.